pkg/kubectl/cmd/util/clientcache.go

```diff
@@ -53,20 +53,18 @@ type ClientCache struct {
 
 	matchVersion bool
 
-	defaultConfigLock sync.Mutex
-	defaultConfig     *restclient.Config
+	mutex         sync.Mutex
```

Comment:
@adamreese thus I think we also need to change that, but we can do it in following.